NRedisTimeSeries/TimeSeriesClientAux.cs

```diff
@@ -110,6 +113,17 @@ private static void AddWithLabels(this IList<object> args, bool? withLabels)
             }
         }
 
+        private static void AddGroupby(this IList<object> args, string groupby, TsReduce? reduce) 
+        {
```

Comment:
I'm not sure if it's necessary, but I'm trying to think about tupling these two variables.
As you can see here, if the user gives only one of these parameters to one of the client functions, the groupby won't work, and no error would arise.